testsuite/forge/src/backend/k8s/swarm.rs

```diff
@@ -331,22 +362,143 @@ pub fn clean_k8s_cluster(
     // parse genesis
     let helm_values = String::from_utf8(raw_helm_values.stdout).unwrap();
     let v: Value = serde_json::from_str(&helm_values).unwrap();
-    let era = &v["genesis"]["era"];
-    let num_validators = &v["genesis"]["numValidators"];
-    let new_era = if era == 1 { 2 } else { 1 };
+    let chain_era = v["genesis"]["era"].as_i64().expect("not a i64") as usize;
+    let curr_num_validators = v["genesis"]["numValidators"].as_i64().expect("not a i64") as usize;
 
-    println!("genesis.era: {} --> {}", era, new_era);
+    // get the new era
+    let mut rng = rand::thread_rng();
+    let new_era = rng.gen::<u32>() as usize;
+    println!("genesis.era: {} --> {}", chain_era, new_era);
```

Comment:
if chain_era here is just only for debugging purpose, whatif we can just do ++ to get new_era instead of using random number above?

Reply:
Random number here is just a stronger guarantee that the chain era is fresh and less likely to be an era we've already seen (and failed).